src/eko/anomalous_dimensions/as4/gqg.py

```diff
@@ -329,18 +331,73 @@ def gamma_qg_nf3(n, sx):
 
 
 @nb.njit(cache=True)
-def gamma_qg_nf2(_n, _sx):
-    return 0
+def gamma_qg_nf1(n, sx):
+    """Implements the part proportional to :math:`nf^1` of :math:`\\gamma_{qg}^{(3)}`.
```

Comment:
blank line after summary line is required in current docstring layout

Reply:
You marked as solved, but you didn't put any newline.

Do we want to start adding [`pydocstyle`](https://github.com/N3PDF/pineko/blob/eccb37285617b9765bdb37cb7d44f815d0769ca2/.pre-commit-config.yaml#L31-L37) @felixhekhorn?

Reply:
we can